src/Tools/dotnet-monitor/Egress/AzureBlob/AzureBlobEgressProvider.cs

```diff
@@ -324,5 +350,23 @@ private static string WrapMessage(string innerMessage)
                 return Strings.ErrorMessage_EgressAzureFailedGeneric;
             }
         }
+
+        private static DefaultAzureCredential CreateManagedIdentityCredentials(string clientId)
+        {
+            var credential = new DefaultAzureCredential(
```

Comment:
nit: Should we be avoiding `var`?

Reply:
The rhs is a ctor so it's very obvious what the type is.